pkg/resources/table_acceptance_test.go

```diff
@@ -25,7 +25,7 @@ func TestAcc_Table(t *testing.T) {
 					resource.TestCheckResourceAttr("snowflake_table.test_table", "name", accName),
 					resource.TestCheckResourceAttr("snowflake_table.test_table", "database", accName),
 					resource.TestCheckResourceAttr("snowflake_table.test_table", "schema", accName),
-					resource.TestCheckResourceAttr("snowflake_table.test_table", "data_retention_days", "1"),
+					resource.TestCheckResourceAttr("snowflake_table.test_table", "data_retention_time_in_days", "1"),
```

Comment:
Do we have any acceptance test showing bad behaviour before and the right one after? I think it would be nice to proceed with bugs by first introducing a test showing the error and then fixing it.

Reply:
its not really bad behavior, its changing the default value of this parameter, as well as changing the name, and adding documentation on how to use this table resource alongside object parameter.